ex/conf.js

```diff
@@ -0,0 +1,48 @@
+exports.config = {
```

Comment:
A to co za plik konfiguracyjny?

Reply:
do skasowania, niepotrzebny jest ten plik, zostal skasowany